scripts/apps/authoring/packages/packages.ts

```diff
@@ -29,7 +29,9 @@ function PackagesCtrl($scope, superdesk, api, search) {
         if (packageItem._type === 'published') {
             superdesk.intent('view', 'item', packageItem);
         } else {
-            superdesk.intent('edit', 'item', packageItem);
```

Comment:
A function is called here to initiate editing. You should go to the implementation and fix the issue there

Reply:
did you mean `superdesk.intent` needs to be changed?

Reply:
`superdesk.intent` is a general function that is used for many things, but yes - you should track the callstack through the intent call until you get to code that is initiating the editing and do the change there.